Misc/NEWS.d/next/Library/2019-02-26-09-31-59.bpo-26228.wyrHKc.rst

```diff
@@ -0,0 +1 @@
+pty.spawn no longer hangs on FreeBSD, OS X, and Solaris.
```

Comment:
This should say "macOS" but I have a follow-up PR for doc updates anyway so I'll fix that there.